spec/rspec/rails/assertion_delegator_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "spec_helper"
+require 'spec_helper'
```

Comment:
This can be removed as `spec_helper` should be required by `.rspec` otherwise please don't change our quotes we don't standardise them and would prefer less git churn.